### DISCUSSION TITLE
신뢰된 봇 더보기

### DISCUSSION THREAD

**Discussion by dazzypark** [koreanbots/koreanbots#648] [제안] [answered]
신뢰된 봇 리스트에도 새로운 봇 리스트처럼 더보기가 있으면 좋겠어요

<img width="2240" alt="Screenshot 2024-07-07 at 01 16 33" src="https://github.com/koreanbots/core/assets/82271684/76411f1b-6834-44f5-a149-153d2bf6f5f5">

**Comment by skinmaker1345** [comment, marked as answer]
제안 감사드립니다. 조사해보겠습니다!

**Comment by skinmaker1345** [comment]
신뢰된 봇을 모아볼 수 있는 페이지가 추가되어 v2.7.0 업데이트에 포함되었습니다.
https://github.com/koreanbots/core/releases/tag/v2.7.0
감사합니다!
